kiwi-desc-saltboot/kiwi-desc-saltboot.spec

```diff
@@ -31,6 +31,12 @@ Provides:       kiwi-boot:saltboot
 Provides:       kiwi-image:cpio
 Provides:       kiwi-image:pxe
 
+%if 0%{?sle_version} >= 120000
+ExclusiveArch:  x86_64 noarch
+%else
```

Comment:
As long as we don't build this for non SLE/openSUSE systems, this is correct :-)

Reply:
This package is only for SLE11 and SLE12. The rest should use dracut-saltboot.